finger-exercises/redux/src/redux/shoppingCart/actions.js

```diff
@@ -0,0 +1,17 @@
+export const actions = {
+  GET_OPEN: '@@SHOPPINGCART/GET_OPEN',
+  GET_CLOSE: '@@SHOPPINGCART/GET_CLOSE'
+};
+
+const actionsCreator = {
+  getOpen: () => ({
+    type: actions.GET_OPEN,
+    payload: { isOpen: true }
```

Comment:
As this action sets `open` field to `true`, its payload is unnecessary.
What if
```js
//actions.js
getOpen: () => ({ type: actions.GET_OPEN })

//redueer.js
switch(  ) {
  case actions.GET_OPEN:
      return { ...state, open: true };
```
The same for the close action.